‘tr_begin’ may be used uninitialized in this function

**Describe the bug**

Compiling EMPIRE fails with these warnings treated as errors:

```
/home/pbmille/repos/EMPIRE/TPL/vt/src/vt/messaging/active.cc: In member function ‘bool vt::messaging::ActiveMessenger::recvDataMsgBuffer(void*, vt::PriorityType, const TagType&, const NodeType&, const bool&, vt::ActionType, vt::RDMA_ContinuationDeleteType)’:
/home/pbmille/repos/EMPIRE/TPL/vt/src/vt/messaging/active.cc:468:40: error: ‘tr_begin’ may be used uninitialized in this function [-Werror=maybe-uninitialized]
             trace::addUserBracketedNote(tr_begin, tr_end, tr_note, trace_irecv);
             ~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/pbmille/repos/EMPIRE/TPL/vt/src/vt/messaging/active.cc: In member function ‘vt::EventType vt::messaging::ActiveMessenger::sendMsgBytes(const NodeType&, const vt::messaging::MsgSharedPtr<vt::messaging::ActiveMsg<vt::messaging::ActiveEnvelope> >&, const MsgSizeType&, const TagType&)’:
/home/pbmille/repos/EMPIRE/TPL/vt/src/vt/messaging/active.cc:229:36: error: ‘tr_begin’ may be used uninitialized in this function [-Werror=maybe-uninitialized]
         trace::addUserBracketedNote(tr_begin, tr_end, tr_note, trace_isend);
         ~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/pbmille/repos/EMPIRE/TPL/vt/src/vt/messaging/active.cc: In member function ‘bool vt::messaging::ActiveMessenger::tryProcessIncomingActiveMsg()’:
/home/pbmille/repos/EMPIRE/TPL/vt/src/vt/messaging/active.cc:763:38: error: ‘tr_begin’ may be used uninitialized in this function [-Werror=maybe-uninitialized]
           trace::addUserBracketedNote(tr_begin, tr_end, tr_note, trace_irecv);
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/pbmille/repos/EMPIRE/TPL/vt/src/vt/messaging/active.cc: In member function ‘vt::messaging::ActiveMessenger::SendDataRetType vt::messaging::ActiveMessenger::sendData(const RDMA_GetType&, const NodeType&, const TagType&)’:
/home/pbmille/repos/EMPIRE/TPL/vt/src/vt/messaging/active.cc:353:36: error: ‘tr_begin’ may be used uninitialized in this function [-Werror=maybe-uninitialized]
         trace::addUserBracketedNote(tr_begin, tr_end, tr_note, trace_isend);
         ~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

**To Reproduce**
Steps to reproduce the behavior:
1. Example/test/snippet of code that fails
2. Compiler, platform, libraries
3. Run command: number of processors, threading options, etc.
4. See error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Platform (please complete the following information):**
 - GCC 7.2.0 on doppio
- vt commit 510cd85 (beta.4)

**Additional context**
Add any other context about the problem here.

--- a/src/vt/messaging/active.cc
+++ b/src/vt/messaging/active.cc
@@ -211,7 +211,7 @@ EventType ActiveMessenger::sendMsgBytes(
 
   {
     #if backend_check_enabled(trace_enabled)
-      double tr_begin;
+      double tr_begin = 0;
       if (ArgType::vt_trace_mpi) {
         tr_begin = vt::timing::Timing::getCurrentTime();
       }
@@ -335,7 +335,7 @@ ActiveMessenger::SendDataRetType ActiveMessenger::sendData(
 
   {
     #if backend_check_enabled(trace_enabled)
-      double tr_begin;
+      double tr_begin = 0;
       if (ArgType::vt_trace_mpi) {
         tr_begin = vt::timing::Timing::getCurrentTime();
       }
@@ -447,7 +447,7 @@ bool ActiveMessenger::recvDataMsgBuffer(
 
       {
         #if backend_check_enabled(trace_enabled)
-          double tr_begin;
+          double tr_begin = 0;
           if (ArgType::vt_trace_mpi) {
             tr_begin = vt::timing::Timing::getCurrentTime();
           }
@@ -742,7 +742,7 @@ bool ActiveMessenger::tryProcessIncomingActiveMsg() {
 
     {
       #if backend_check_enabled(trace_enabled)
-        double tr_begin;
+        double tr_begin = 0;
         if (ArgType::vt_trace_mpi) {
           tr_begin = vt::timing::Timing::getCurrentTime();
         }

